backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplication.java

```diff
@@ -186,6 +191,15 @@ protected void createWakeLock (boolean use) {
 		}
 	}
 
+	@TargetApi(28)
+	private void tryRenderUnderCutout(boolean render) {
+		if(render && getVersion() >= Build.VERSION_CODES.P) {
+			WindowManager.LayoutParams lp = getWindow().getAttributes();
+			lp.layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES;
+			getWindow().setAttributes(lp);
```

Comment:
Haven't tested but do we need this last line?

Reply:
You are right, I normally use it with the line